src/interfaces.ts

```diff
@@ -123,10 +118,25 @@ export interface Connection {
   contract: string;
   list: () => Promise<TableMetadata[]>;
   create: (
-    query: string,
-    options?: CreateTableOptions
+    /** The ChainId of the chain this table should be created on.
+     *  MUST match the chain that the SDK connected to
+     **/
+    chainId: number,
+    /** The schema that defines the columns and constraints of the table,
+     *  e.g.
```

Comment:
The example here isn't a string. I think you'll want to use real JS formatting so folks could (in theory) copy paste this code, because that's what they'll end up doing.